auth-web/src/plugins/vuetify.ts

```diff
@@ -53,7 +53,8 @@ export default new Vuetify({
         },
         navMenuBg: {
           base: '#26527d'
-        }
+        },
+        anchor: '#1A5A96'
```

Comment:
Why not use the existing Vuetify named colour "bcgovblueLink"  (line 46) ?

Reply:
I can't see where this named colour is used. If it isn't, please delete it.

Reply:
Because anchor defines this globally for anchors.

Reply:
I can't find anywhere that `color="anchor"` is used in our code. Are you saying the Vuetify components use this?

Reply:
Jer said:
>“By default, the theme service will use your application’s primary color for anchor tags. You can override this by adding an anchor property to the theme:”
>
>light: {
>primary: ‘#3f51b5’,
>secondary: ‘#b0bec5’,
>anchor: ‘#8c9eff’,
>},
>
>as the example from the vuetify docs

So, yes, it is a named colour that is, presumably, used internally in some Vuetify components. Ya learn somethin' new every day!